deployment/live/gcp/conformance/terragrunt.hcl

```diff
@@ -3,11 +3,15 @@ terraform {
 }
 
 locals {
-  env        = path_relative_to_include()
-  project_id = get_env("GOOGLE_PROJECT", "trillian-tessera")
-  location   = get_env("GOOGLE_REGION", "us-central1")
-  base_name  = get_env("TESSERA_BASE_NAME", "${local.env}-conformance")
-  log_origin = "conformance-gcp-${local.env}"
+  env                          = path_relative_to_include()
+  project_id                   = get_env("GOOGLE_PROJECT", "trillian-tessera")
+  location                     = get_env("GOOGLE_REGION", "us-central1")
+  base_name                    = get_env("TESSERA_BASE_NAME", "${local.env}-conformance")
+  conformance_gcp_docker_image = "us-central1-docker.pkg.dev/trillian-tessera/docker-${local.env}/conformance-gcp:latest"
```

Comment:
Should the docker image refer to the location above instead of `us-central1`?

Reply:
`${var.region}`

Reply:
Possibly... updated to do that for now.